azext_iot/product/test/command_tests.py

```diff
@@ -278,9 +278,8 @@ def _read_certificate_from_file(certificate_path):
     with open(file=certificate_path, mode="rb") as f:
         data = f.read()
 
-        from base64 import encodestring  # pylint: disable=no-name-in-module
-
-        return encodestring(data)
+        from base64 import b64encode  # pylint: disable=no-name-in-module
+        return b64encode(data).decode()
```

Comment:
nit: It would be good to explicitly state the encoding argument for `decode()` for faster readability (though utf8 is default).